src/Data/Swagger/Internal/ParamSchema.hs

```diff
@@ -193,6 +193,12 @@ timeParamSchema fmt = mempty
 instance ToParamSchema Day where
   toParamSchema _ = timeParamSchema "date"
 
+-- |
+-- >>> toParamSchema (Proxy :: Proxy TimeOfDay) ^. format
+-- Just "hh:MM:ss"
+instance ToParamSchema TimeOfDay where
+  toParamSchema _ = timeParamSchema "hh:MM:ss"
```

Comment:
Why are `hh`, `ss` small, but `MM` is capitalized?  Could you explain this in the haddocks?

Reply:
Actually I am not sure if it makes a difference if MM is capitalized or not. I just took the schema that was used for LocalTime ("yyyy-mm-ddThh:MM:ss") and removed the parts I didn't need.
So basically I left the M capitalized for being consistent, but I don't know if there is another reason.

Reply:
`mm` is month. `MM` is minutes.